codegen/rename_bs_module.py

```diff
@@ -34,7 +34,13 @@ def fix_module(filepath):
         content = content.replace("Regex", "_regex")
         print("Replaced", typeName, newTypeName)
         content = content.replace(typeName, newTypeName)
-    newFile.write_text(content)
+    newFile.write_text(
+        content.replace(
+            # Fix timestamp message encoding which is a rfc3339 string, not an object
+            'timestamp" (Js.Json.object_',
+            'timestamp" (Js.Json.string',
```

Comment:
So we keep the protoc python encoding for timestamp and adapt the rest ?

Reply:
Yes, the code generated by the protobuf default compiler does a special treatment for timestamp, so we only have to adapt rescript and haskell encoders.